events/Lambda/AddPermission20150331v2.json

```diff
@@ -9,6 +9,35 @@
     "mitreAttackTechniques": [
         "T1098 - Account Manipulation"
     ],
+    "mitreAttackSubTechniques": [
+
+    ],
+    "unverifiedMitreAttackTechniques": [
+        {
+            "technique": "T1098 - Account Manipulation",
+            "reason": "The AddPermission API call can be used to alter permissions, effectively manipulating accounts to maintain access or escalate privileges."
+        },
+        {
+            "technique": "T1070 - Indicator Removal",
+            "reason": "Using AddPermission to adjust access rights can help hide malicious activities by removing or altering indicators of compromise."
```

Comment:
I don't see it as strictly related, especially that here you allow somebody else to execute the lambda.

Reply:
Removed.